### DISCUSSION TITLE
0.53.0 rc15 files upload size problem

### DISCUSSION THREAD

**Discussion by itiloo** [rejetto/hfs#664] [General]
Hi Rejetto:
In version 0.53.0 rc14 all works perfectly.
I upgrade to 0.53.0 rc15 and it dont permit upload files with "file too big" error.

![image](https://github.com/rejetto/hfs/assets/94358936/45f1f9eb-5810-4646-94d4-43514a71895d)

The file have 20K. Does version rc15 have any size control implemented?.

Thank you so much Rejetto.

**Comment by PH5HP** [comment]
Hi,

Just tested it also, but no problems with uploading here...

Kind regards,
Henk - fileserver.ph5hp.nl - ph5hp.nl

**Reply by itiloo** [reply]
Something changes when you go up to version rc 15, because it doesn't happen in rc 14 and it happens when you go up to rc15 and rc16.

**Reply by PH5HP** [reply]
Hi,

I have tested it with rc16 with no problem .. You may try it yourself. 

https://fileserver.ph5hp.nl 

and try to upload in the upload folder...

Kind regards,
Henk - fileserver.ph5hp.nl - ph5hp.nl

**Reply by rejetto** [reply]
ok so there's something different in his case that triggers the problem, but i think it's a bug nonetheless

**Comment by rejetto** [comment]
did you try to go back to rc14?

**Reply by itiloo** [reply]
Yes, that's why I know it's failing me starting with rc15. I go back to rc14 and everything is perfect, I go up to rc15 or rc16 and I fail.

**Comment by rejetto** [comment]
there is only [one change in rc15](https://github.com/rejetto/hfs/commit/4eda08f94204822cbd95cd7b05be78c82ea49c5e) that can affect upload in such way, but i still fail to see how.
tell me more about the disk you are uploading to.

**Reply by rejetto** [reply]
i was wrong, the change that may have a broken [is another](https://github.com/rejetto/hfs/commit/787224a3d6c7bdc0737497b2a4e182b415d4acbf).
but i cannot just remove the change, as it's a fix for another problem.

**Comment by rejetto** [comment]
are you on Windows?

**Reply by itiloo** [reply]
Hi Rejetto, no, linux

**Comment by rejetto** [comment]
can you report here the output of this command? `df -k`
and also tell me what is the path on disk of the folder you are trying to upload to

**Reply by itiloo** [reply]
![image](https://github.com/rejetto/hfs/assets/94358936/e34821ec-c4a5-44df-b88b-d5fd707de711)


Space /root/.hfs/espacios/ivan is the virtual file system

**Comment by rejetto** [comment]
ok thanks

**Comment by rejetto** [comment]
are you sure you specified parameter -k ?

**Reply by rejetto** [reply]
i think i see the problem, and it's like your df is not recognizing parameter -k.
What linux is yours?

**Reply by itiloo** [reply]
I do df -h sorry.
Redhat

**Comment by rejetto** [comment]
please report the result of command `df -k /root/.hfs/espacios/ivan`

**Reply by itiloo** [reply]
![image](https://github.com/rejetto/hfs/assets/94358936/b3eae05e-2897-4ae3-8e4d-dbcc7c3692ce)

**Reply by itiloo** [reply]
This is a nfs space

**Comment by rejetto** [comment]
ok, i think i fixed it.
do you run hfs with npx or executable?

**Reply by itiloo** [reply]
executable

**Comment by rejetto** [comment]
ok, can you confirm this version fixes your problem?
go admin-panel > home > right-click on check-for-updates
https://github.com/user-attachments/files/16065424/hfs-linux-x64-0.53.0-rc16.3.zip

**Reply by itiloo** [reply]
Yes Rejetto, in that version works perfectly. Thank you so much. What was the problem?

**Reply by rejetto** [reply]
i recently had to rewrite the parsing of the output of "df", but introduced an error

**Reply by itiloo** [reply]
Perfect, then I understand that it worked for PH5HP because it used Windows.

**Reply by itiloo** [reply]
I install rc18 and works perfectly Rejetto

**Reply by rejetto** [reply]
thank you for your feedback!
